website/docs/feature-flag-tutorials/python/examples.md

```diff
@@ -0,0 +1,508 @@
+
+
+![Edit your gradual rollout strategy by selecting the 'edit strategy' button.](/img/python-ex-edit-strategy.png)
+
+
+This will take you to the gradual rollout form. Next, add a new constraint.
+
+
+![Add a constraint to your strategy.](/img/python-ex-add-constraint.png)
+
+Let’s say we are experimenting with releasing the “delete” feature to all environments except production to test it with internal users before releasing it to all users.
+
+![The new constraint form includes a context field, operator, and values field to customize the conditions under which a user will be exposed to the flag](/img/python-ex-constraint-page.png)
+
+
+We can configure the constraint in the form to match these requirements:
+- The context field is set to **environment**
+- The operator is set to **NOT_IN**
+- The value added is **production**
+
+
+Once you’ve filled out the proper constraint fields, select ‘Done’ and save the strategy.
+
+![Once you add a constraint, you can see it listed underneath your strategy in Unleash.](/img/python-ex-constraint-added.png)
+
+
+Your release process is now configured with an environment-dependent strategy constraint.
+
+Alternatively, you can send an API command to apply the same requirements:
+
+
+```py
+import requests
+import json
+
+url = "<your-unleash-url>/api/admin/projects/:projectId/features/:featureName/environments/:environment/strategies/:strategyId"
+
+payload = json.dumps({
+  "name": "flexibleRollout",
+  "disabled": False,
+  "constraints": [
+    {
+      "values": [
+        "production"
+      ],
+      "inverted": false,
+      "operator": "NOT_IN",
+      "contextName": "environment",
+      "caseInsensitive": false
+    }
+],
+  "variants": [],
+  "parameters": {
+    "groupId": "delete_survey_flag",
+    "rollout": "50",
+    "stickiness": "sessionId"
+  },
+  "segments": []
+})
+
+headers = {
+  'Content-Type': 'application/json',
+  'Accept': 'application/json',
+  'Authorization': '<API_KEY_VALUE>'
+}
+
+response = requests.request("PUT", url, headers=headers, data=payload)
+
+print(response.text)
+```
+
+> Note:
+> If you already have a gradual rollout strategy for your flag, use a PUT request to update it.
+> If you’re creating a new strategy, use a POST request.
+
+Check out our [API docs on updating flag strategies](/reference/api/unleash/update-feature-strategy) to learn more.
+
+Read our documentation for more context on [strategy constraint configurations](/reference/strategy-constraints) and use cases.
+
+
+## Server-side A/B Testing in Python
+
+
+A/B testing is a common way for teams to test out how users interact with two or more versions of a new feature that is released. Server-side A/B testing can help with making infrastructure improvements and comparing different versions of server-side methods. At Unleash, we call these flag [variants](/reference/feature-toggle-variants).
```

Comment:
Related to https://github.com/Unleash/unleash/pull/6094/files#r1474849656 - I much prefer the wording in this paragraph!